src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import App from './components/App';
```

Comment:
This ones on me, `App.js` should be in the `/src` folder, you typically want all of your top level files directly in `/src`. Not much else goes here, so we should be good after you revert this. Sorry!